src/codegen_call_native.cpp

```
@@ -0,0 +1,178 @@
+        std::string rtName = deriveRuntimeFnName(package, rtSuffix)
+            + std::to_string(n);
+        auto *fnTy = llvm::FunctionType::get(ptrTy_, argTypes, false);
+        auto fn = mod_->getOrInsertFunction(rtName, fnTy);
+        return builder_.CreateCall(fn, args, entry->fn_name);
+    }
+
+    // --- Normal path ---
+    requireArgs(e, static_cast<size_t>(entry->arity));
+
+    // Look up NativeFnSignature for type info
+    std::string sigKey = nativeSigKey(package, e.callee);
```

Clang (14.0.6)

[warning] variable 'sigKey' is not initialized

(cppcoreguidelines-init-variables)